runtime/compiler/runtime/Runtime.cpp

```diff
@@ -1008,26 +1008,25 @@ void initializeCodeRuntimeHelperTable(J9JITConfig *jitConfig, char isSMP)
 
 #if (defined(TR_HOST_X86) || defined(TR_HOST_POWER) || defined(TR_HOST_S390)) && defined(J9VM_JIT_NEW_DUAL_HELPERS)
    SET(TR_newObject,                  (void *)jitNewObject,              TR_CHelper);
-   SET(TR_newValue,                   (void *)jitNewValue,              TR_CHelper);
    SET(TR_newArray,                   (void *)jitNewArray,               TR_CHelper);
    SET(TR_aNewArray,                  (void *)jitANewArray,              TR_CHelper);
 
-   SET(TR_newObjectNoZeroInit,        (void *)jitNewObjectNoZeroInit, TR_CHelper);
-   SET(TR_newValueNoZeroInit,        (void *)jitNewValueNoZeroInit, TR_CHelper);
-   SET(TR_newArrayNoZeroInit,         (void *)jitNewArrayNoZeroInit,  TR_CHelper);
-   SET(TR_aNewArrayNoZeroInit,        (void *)jitANewArrayNoZeroInit, TR_CHelper);
+   SET(TR_newObjectNoZeroInit,        (void *)jitNewObjectNoZeroInit,    TR_CHelper);
+   SET(TR_newArrayNoZeroInit,         (void *)jitNewArrayNoZeroInit,     TR_CHelper);
+   SET(TR_aNewArrayNoZeroInit,        (void *)jitANewArrayNoZeroInit,    TR_CHelper);
 #else
    SET(TR_newObject,                  (void *)jitNewObject,              TR_Helper);
-   SET(TR_newValue,                  (void *)jitNewValue,              TR_Helper);
    SET(TR_newArray,                   (void *)jitNewArray,               TR_Helper);
    SET(TR_aNewArray,                  (void *)jitANewArray,              TR_Helper);
 
-   SET(TR_newObjectNoZeroInit,        (void *)jitNewObjectNoZeroInit, TR_Helper);
-   SET(TR_newValueNoZeroInit,        (void *)jitNewValueNoZeroInit, TR_Helper);
-   SET(TR_newArrayNoZeroInit,         (void *)jitNewArrayNoZeroInit,  TR_Helper);
-   SET(TR_aNewArrayNoZeroInit,        (void *)jitANewArrayNoZeroInit, TR_Helper);
+   SET(TR_newObjectNoZeroInit,        (void *)jitNewObjectNoZeroInit,    TR_Helper);
+   SET(TR_newArrayNoZeroInit,         (void *)jitNewArrayNoZeroInit,     TR_Helper);
+   SET(TR_aNewArrayNoZeroInit,        (void *)jitANewArrayNoZeroInit,    TR_Helper);
 #endif
 
+   SET(TR_newValue,                   (void *)jitNewValue,               TR_CHelper);
+   SET(TR_newValueNoZeroInit,         (void *)jitNewValueNoZeroInit,     TR_CHelper);
```

Comment:
Use `TR_CHelper` for `jitNewValue` and `jitNewValueNoZeroInit` regardless of `J9VM_JIT_NEW_DUAL_HELPERS`, because there are no definitions of `old_fast_jitNewValue`/`old_fast_jitNewValueNoZeroInit()` functions in cnathelp.cpp.

Reply:
See #8160 (arm) and #8161 (aarch64).
Those helpers are explicitly defined by `NEW_DUAL_MODE_HELPER` instead of `DUAL_MODE_HELPER` to avoid "undefined symbol" errors for missing `old_fast_*` functions.